views/translations.html.haml

```diff
@@ -0,0 +1,38 @@
+- c = 0
+- if !keys.empty?
+  %form.form-horizontal{ action: "#{root_path}update", method: 'post', onsubmit: 'disableSubmit(event.target);' }
+    - filter_params.each do |filter, value|
+      %input{name: "filters[#{filter}]", type: 'hidden', value: value}
+    - keys.each do |key|
+      - c = c + 1
+      .form-group{ class: c == 1 ? 'first_key' : '' }
+        - json_key = store.json_key?(key)
+        %label.col-sm-3.control-label.translation_label{ class: store.key_complete?(key) ? 'incomplete' : '' }
+          = key.name
+        .col-sm-7
+          %table
+            - translation_repository.all_for_key(key).each do |translation|
+              %tr
+                %td
+                  = "#{translation.locale_id}:"
+                %td
+                  - if json_key
+                    %span [JSON]
+
+                    %textarea.form-control{ cols: 80, rows: translation.number_of_lines, name: "translations[#{translation.id}][value]", dirname: 'auto' }= translation.value.to_json
```

Comment:
using `dirname` does not show the text in Right-to-Left form (for RTL languages).